profile.html

```diff
@@ -6,12 +6,13 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Quiz App</title>
     <link rel="stylesheet" href="profile.css" />
+    <script src="index.js"></script>
   </head>
   <body>
     <header>
       <h1>QUIZ APP</h1>
     </header>
-    <main>
+    <main data-js="main">
       <section class="img">
         <img class="imgprofile" alt="Profilbild" src="./assets/profile.png" />
```

Comment:
```suggestion
        <img class="imgProfile" alt="Profilbild" src="./assets/profile.png" />
```
oder vielleicht willst du ja sowieso nochmal ändern wegen BEM :)

Reply:
hab noch keine BEM durchgeführt , aber danke für die Tipps